static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -403,6 +430,24 @@ export class DeprecatedLine extends Component<Props, State> {
                 </SourceMapDebuggerModalButton>
               </Fragment>
             ) : null}
+            {showSentryAppStacktraceLink && (
```

Comment:
If the sentry app components don't have a loading state I think they should probably be to the right side so the position doesn't move around as the github/codecov links pop in.